lib/elixir/lib/dynamic_supervisor.ex

```diff
@@ -215,10 +215,7 @@ defmodule DynamicSupervisor do
           extra_arguments: [term()]
         }
 
-  @typedoc "Options given to `start_link` functions"
-  @type option :: GenServer.option()
```

Comment:
This now-removed type was being used at https://github.com/elixir-lang/elixir/blob/main/lib/elixir/lib/task/supervisor.ex#L77C1-L77C37 which now needs to be updated

Reply:
done and thanks:
https://github.com/elixir-lang/elixir/pull/13704